packages/freezed/README.md

```diff
@@ -1229,6 +1230,44 @@ class MyModel with _$MyModel {
 }
 ```
 
+### classes with generic argument factories
+
+If you annotate your constructor(s) with `@JsonSerializable(genericArgumentFactories: true)`, [Freezed] will
+require a `fromJson` and `toJson` function for each generic type. For example, the following class requires an aditional `fromJsonT` and `toJsonT`:
```

Comment:
I don't see an example for the `toJsonT`

Reply:
I've separated the fromJson and toJson examples